package.json

```diff
@@ -1,20 +1,37 @@
 {
-  "name": "holochain_client",
+  "name": "@holochain/hc-web-client",
   "version": "0.0.1",
-  "description": "Holochain websocket interface",
-  "main": "src/index.js",
+  "description": "Holochain container connection helper for web based UIs",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
   "author": "Michael Dougherty <michael.dougherty@holo.host>",
```

Comment:
Apparently there is a `contributors` field which would be more appropriate at this point https://docs.npmjs.com/files/package.json#people-fields-author-contributors